### PR TITLE
Add WinWhisper 1.2.0

### PR DESCRIPTION
**Pull request by GewoonJaap**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124574)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @GewoonJaap,

Unfortunately, the package didn't pass a Defender scan:
image

This might be a false positive and we can rescan tomorrow.

**Comment by GewoonJaap** [comment]
Owh Huh. Yeah this must be a false positive, the program is open source: https://github.com/GewoonJaap/WinWhisper

It's basically a console front-end for Whisper.NET with some added features and definitely no malware.

If you need any info from me, please let me know. I would love to cooperate to get this issue fixed.

**Comment by GewoonJaap** [comment]
https://www.virustotal.com/gui/file-analysis/OGE5OWYxOWM2MWE4ZWUyOGEyMGMwMGU4ZTUwYjczODA6MTY5ODk2MDUzOQ==

Looks like some AVs detect it as a Trojan? I think I should submit a report to those AVs so they can re-evaluate it?

**Comment by GewoonJaap** [comment]
@stephengillie I just reported the setup as a false postive to Trend Micro and Rising. Is there anything else I can do?

**Comment by stephengillie** [comment]
Hi @GewoonJaap,

Thanks for your patience. The Defender check passed. But this package doesn't seem to add any files to the file system. Is this expected?

**Comment by GewoonJaap** [comment]
> Hi @GewoonJaap,
> 
> Thanks for your patience. The Defender check passed. But this package doesn't seem to add any files to the file system. Is this expected?

No this isn't expected. It should install to appdata. I and other people executed the .exe themselfs and that worked. 
I just tested it on a different PC with the /s flag and it installs perfectly fine in silence

**Comment by stephengillie** [comment]
Running without the silent switch gives a dependency:
image

File: manifests/j/Jaap/WinWhisper/1.2.0/Jaap.WinWhisper.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Jaap.WinWhisper
+PackageVersion: 1.2.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /s
```
Comment (stephengillie) [inline manifests/j/Jaap/WinWhisper/1.2.0/Jaap.WinWhisper.installer.yaml:8]:
```suggestion
  Silent: /s
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```

**Comment by GewoonJaap** [comment]
> Running without the silent switch gives a dependency: image

Ahh makes sense, thanks for helping me out with this :)

**Comment by stephengillie** [comment]
Misread one dependency. Fixing it leaves the other - `Microsoft.VCRedist.2022.x64` - which I believe is not yet in our repository, as the only "2022" I see is `Microsoft.VCRedist.2022.arm64`. 
image

**Comment by GewoonJaap** [comment]
> Misread one dependency. Fixing it leaves the other - `Microsoft.VCRedist.2022.x64` - which I believe is not yet in our repository, as the only "2022" I see is `Microsoft.VCRedist.2022.arm64`. image

How should I proceed? Is there a way to add this dependency to the repo? Because if we can't configure this as a required dependency, the installer might fail for some users, right?

**Comment by stephengillie** [comment]
@GewoonJaap, 
Yes, the dependency will need to have a manifest added before this package can be approved.

**Pull request closed without merging**